tests/test_ml.py

```diff
@@ -45,17 +43,19 @@
     # "accuracy",
 ]
 
+predict_files = [
+    (DATA_PATH/'GCA_000692775_1_trunc2.fna.gz', ),
+    (DATA_PATH/'GCA_000692775_1_trunc2.faa.gz', ),
+    (DATA_PATH/'GCA_000692775_1_trunc2.fna.gz', DATA_PATH/'GCA_000692775_1_trunc2.faa.gz')
+]
```

Comment:
Do we now only test gzipped files? Both gzipped and plaintext should be covered by tests. Sorry, if my previous comment was too vague.